kubetest/kubeadmdind/kubeadm_dind.go

```diff
@@ -402,7 +402,7 @@ func (d *Deployer) clusterSize() (int, error) {
 	return 0, nil
 }
 
-// Create a local log artifacts directory
+// makeLogDir create a local log artifacts directory
```

Comment:
Grammar nit: I think this should be creates i.e. `makeLogDir creates a local ....`